sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -642,3 +642,14 @@ def test_issue_8601():
     assert str(c1) == '6.93334070531408e-5'
     c2 = catalan(-35.4).evalf()
     assert str(c2) == '-4.14189164517449e-24'
+
+
+def test_motzkin():
+    assert motzkin.find_motzkin_numbers_in_range(100, 10000) == [127, 323, 835, 2188, 5798]
+    assert motzkin.find_first_n_motzkin(7) == [1, 1, 2, 4, 9, 21, 51]
+    assert motzkin.is_motzkin(51) == True
+    assert motzkin.is_motzkin(25) == False
+    assert motzkin.is_motzkin(1) == True
+    assert motzkin.find_motzkin_numbers_in_range(-10, 40) == [1, 1, 2, 4, 9, 21]
+    assert motzkin.motzkin(1) == 1
+    assert motzkin.motzkin(8) == 127
```

Comment:
You should probably test some incorrect inputs as well, as 

```
motzkin.is_motzkin(1.5)
x = Symbol('x')
motzkin.is_motzkin(x)
motzkin.is_motzkin('foo')
```
and so on just to see that nothing unexpected happens. If an exception is thrown you can use something like
`raises(ExceptionName, lambda: motzkin.bad_expression())`